### PR TITLE
[test] fix base-config-fonturls.js on NodeJS < 8

### PR DESCRIPTION
**Pull request by pkra**
Make version detection use MathJax itself, rather than the package-lock file.

### CODE REVIEW

**Comment by pkra** [comment]
Something is wrong with travis. I'll restart it.

**Comment by pkra** [comment]
Huh. One travis run completely works, the other completely fails. I'm not sure what's happening here.

**Comment by pkra** [comment]
Ah, ok. The commit itself is passing but the merged PR is not.

File: test/base-config-fonturl.js [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
 tape('basic configuration: check fontURL', function (t) {
     t.plan(2);
-
-    var tex = 'a';
+    
+    mjAPI.config({
+        extensions: '[test]/version.js',
+        paths: {test: path.join(__dirname,'assets/')}
+    });
```
Comment (pkra) [inline test/base-config-fonturl.js]: I think `mjAPI.start()` is needed to restart after the configuration to avoid tape failures.

**Comment by dpvc** [comment]
I'm closing this one, since PR #405 takes care of it better.

**Pull request closed without merging**